### ISSUE TITLE
Pdfjs loader issue within app generated using electron-create-react-app

### ISSUE THREAD

**Issue by sid-thakur** [wojtekmaj/react-pdf#723]
**Describe the bug**
Issue loading pdfjs within app generated using [electron-create-react-app](https://github.com/csepulv/electron-with-create-react-app). The error states that an appropriate loader to handle the file type is needed. 

**To Reproduce**
Steps to reproduce the behavior including code snippet (if applies):
1. clone repo https://github.com/csepulv/electron-with-create-react-app
2. install dev dependencies and react-pdf
3. add sample demo code (below)
4. run via `npm run start`

```
import React, { useState } from "react";
import { Document, Page, pdfjs } from "react-pdf";
import "react-pdf/dist/esm/Page/AnnotationLayer.css";

pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;

const pdf = require("./doc.pdf");

function MyApp0() {
  const [numPages, setNumPages] = useState(null);
  const [pageNumber, setPageNumber] = useState(1);

  function onDocumentLoadSuccess({ numPages }) {
    setNumPages(numPages);
  }

  return (
    <div>
      <Document file={pdf.default} onLoadSuccess={onDocumentLoadSuccess}>
        {Array.from(new Array(numPages), (el, index) => (
            <Page 
                key={`page_${index + 1}`} 
                pageNumber={index + 1}
            />
        ))}
      </Document>
    </div>
  );
}

export default MyApp0;
```

**Expected behavior**
PDF file is loaded as shown in sample examples

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: FireFox
 - React-pdf version: 5.1.0

```
8:52:34 PM react.1    |  Failed to compile.
8:52:34 PM react.1    |  Error in ./~/react-pdf/~/pdfjs-dist/build/pdf.js
8:52:34 PM react.1    |  Module parse failed: C:\Users\sidth\Documents\project\node_modules\react-pdf\node_modules\pdfjs-dist\build\pdf.js Unexpected token (447:8)
8:52:34 PM react.1    |  You may need an appropriate loader to handle this file type.
8:52:34 PM react.1    |  SyntaxError: Unexpected token (447:8)
8:52:34 PM react.1    |   @ ./~/react-pdf/dist/umd/entry.js 30:36-57

8:52:34 PM react.1    |  Error in ./~/react-pdf/~/pdfjs-dist/lib/web/ui_utils.js
8:52:34 PM react.1    |  Module parse failed: C:\Users\sidth\Documents\project\node_modules\react-pdf\node_modules\pdfjs-dist\lib\web\ui_utils.js Unexpected token (112:8)
8:52:34 PM react.1    |  You may need an appropriate loader to handle this file type.
8:52:34 PM react.1    |  SyntaxError: Unexpected token (112:8)
8:52:34 PM react.1    |   @ ./~/react-pdf/dist/umd/eventBus.js 8:16-54
```

**Comment by sid-thakur** [comment]
Still no luck figuring out the above issue, but I have learned that same problem has been posted in a number of threads. Here's my setup: electron-react-app as in post above, with following libraries
```
`-- react-pdf@5.1.0
  +-- pdfjs-dist@2.5.207
  `-- worker-loader@3.0.8
```
Here's the result in electron window:
```
Failed to compile.

Error in ./~/pdfjs-dist/build/pdf.js
Module parse failed: C:\Users\sidth\Documents\Research_work\personal_info_mgmt\pico-10-csepul-era\node_modules\pdfjs-dist\build\pdf.js Unexpected token (447:8)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (447:8)
 @ ./~/react-pdf/dist/umd/entry.js 30:36-57
```

Same setup as above works if I use `react-pdf` in a CRA (create react app) application, but it doesn't seem to work with electron-CRA. 

Based on the elegant solution proposed in [https://github.com/yurydelendik/pdfjs-react](https://github.com/yurydelendik/pdfjs-react) @yurydelendik I discovered that if I downgrade to `pdfjs-dist@1.8.432` and `worker-loader@0.8.1` then the electron app can load and show PDF files using pdfjs-dist. I'd like to use latest libraries and get over the above issue with missing worker-loader.
Thanks!
